overlord/state/state.go

```diff
@@ -483,7 +483,7 @@ func (s *State) Prune(pruneWait, abortWait time.Duration, maxReadyChanges int) {
 			if spawnTime.Before(pruneLimit) && len(chg.Tasks()) == 0 {
 				chg.Abort()
 				delete(s.changes, chg.ID())
-			} else if spawnTime.Before(abortLimit) {
+			} else if spawnTime.Before(abortLimit) && spawnTime.After(startOfOperation) {
```

Comment:
what we want I think is to replace replace spawnTime with startOfOperation if spawnTime is before startOfOperation, we still want to abort that change in the worst case but based on when we actually started running

Reply:
I see your point, makes sense,thanks.

Reply:
Done, let me know if it makes sense now.